### PR TITLE
Add a simple test and Travis CI config.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - python: 3.6
+    - python: 3.7
+    - python: 3.8
+    - python: nightly
+  allow_failures:
+    - python: nightly
+
+install:
+  - pip install -r requirements.txt
+  - pip install -r requirements-dev.txt
```
Comment (mrakitin) [inline .travis.yml:20]: Maybe here we can run the script with `-h`? Such as:
```bash
python resource_health_check.py -h
```

**Review by mrakitin — COMMENTED** [review]
Looks great! Left a single suggestion.

File: .travis.yml (hunk 1 above)
Comment (danielballan) [reply .travis.yml:20]: Good idea. I did it in `test.py` via `subprocess.check_call` so that all the tests are in one place.

Comment (mrakitin) [reply .travis.yml:20]: Sounds great! I like the idea.

**Pull request merged**